Oneirocom/Magick
# Fixed multiple loading screen when more than one tab are opened

## What Changed:
when loading the spell, show the loading screen only on the active tab

## How to test:

- Open multiple spells and try to refresh the page, you might not see it with fast internet.

## Additional information:

Any other information that might be useful while reviewing.
